Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonButton.cs

```diff
@@ -817,7 +817,7 @@ protected override void OnMouseDown(MouseEventArgs e)
         /// <inheritdoc />
         protected override void OnMouseUp(MouseEventArgs e)
         {
-            if (!Values.ShowSplitOption || ContextMenuStrip is not { Visible: true })
+            if (!Values.ShowSplitOption || ContextMenuStrip is { Visible: true })
```

Comment:
Hold on.....
 `Not = true` 
should be shortened to `== false`
There's a reason ReSharper did not recommend the change.
Please back it out.

Reply:
Reverted changes